docs/setup/install/troubleshooting/index.md

```diff
@@ -12,9 +12,13 @@ menu_name: docs_{{ .version }}
 section_menu_id: setup
 ---
 
+# Troubleshooting Kubestash
+
+This doc is about kubestash installation troubleshooting.
+
```

Comment:
# Troubleshooting KubeStash Installation

This guide will show you how to troubleshoot some common KubeStash installation issues.